### DISCUSSION TITLE
PrimeVueResolver() not working properly

### DISCUSSION THREAD

**Discussion by sidjoshi001** [primefaces/community#1304] [PrimeVue] [answered]
I am using  `PrimeVueResolver()` in vite configuration for auto import components. It's working fine but however 

```
Components({
            deep: true,
            dirs: [
                './resources/ts/components'
            ],
            resolvers: [
                PrimeVueResolver()
            ]
        }),
```

```
IconField, InputIcon, InputGroup 
```

not working with this auto-import..!! 

### Tell me how to resolve this

**Comment by tugcekucukoglu** [comment, marked as answer]
It seems we should wait for the new release. This PR is still waiting for the merge https://github.com/unplugin/unplugin-vue-components/pull/731
